### PR TITLE
Add Slack environment variables

### PR DESCRIPTION
**Pull request by andyhd**
Add `SLACK_API_TOKEN` and `SLACK_CHANNEL` environment variables

### CODE REVIEW

File: charts/cpanel/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+## [2.3.4] - 2019-11-11
```
Comment (xoen) [inline charts/cpanel/CHANGELOG.md]: I'd say it's more of a `2.4.0` version considering "it adds a feature in a retro-compatible way".

File: charts/cpanel/values.yaml [hunk 2]
```diff
@@ -40,9 +41,10 @@ secretEnv:
   RSTUDIO_AUTH_CLIENT_DOMAIN: ""
   RSTUDIO_AUTH_CLIENT_ID: ""
   RSTUDIO_AUTH_CLIENT_SECRET: ""
+  SECRET_KEY: ""
```
Comment (xoen) [inline charts/cpanel/values.yaml]: I like alphabetical order 👍

**Comment by andyhd** [comment]
What missing CHANGELOG entry?

File: charts/cpanel/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+## [2.4.0] - 2019-11-11
+### Changed
+- Added Slack environment variables.
```
Comment (xoen) [inline charts/cpanel/CHANGELOG.md]: 👍

**Pull request merged**